src/core/src/op/space_to_batch.cpp

```diff
@@ -102,7 +102,7 @@ bool ngraph::op::v1::SpaceToBatch::evaluate_space_to_batch(const HostTensorVecto
 
     auto data_shape = data->get_shape();
 
-    if (!(data->get_shape().size() == 4 || data->get_shape().size() == 5)) {
+    if (!(data->get_shape().size() == 3 || data->get_shape().size() == 4 || data->get_shape().size() == 5)) {
```

Comment:
@jane-intel Do we really need this check? Can we use it for other ranks?

Reply:
I don't think so. As you can see [here](https://docs.openvino.ai/2023.0/openvino_docs_ops_movement_SpaceToBatch_2.html) -- the input could be nD, n>=2